packages/pubsub/lib/event.ts

```diff
@@ -1,8 +1,8 @@
 import type { DBTeam, DBUser } from '@nangohq/types';
 
-type JsonValue = string | number | boolean | null | JsonValue[] | { [key: string]: JsonValue } | Record<string, any>;
+type Serializable = string | number | boolean | Date | null | undefined | Serializable[] | { [key: string]: Serializable };
```

Comment:
events are serialized via node:v8 serialiaze. I thought initially we will do json encoding but never modified this type accordingly